### PR TITLE
JY1, JY2, JY3

### PR DESCRIPTION
**Pull request by ciaua**
Dear organizers,

We would like to submit our result of source separation for the SiSEC this year. There are results of three models in this pull request.

### CODE REVIEW

File: submissions/JY1/description.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+
+## Method
+
+Denoising auto-encoder with skip connections. It shares some similarities with the U-Net proposed by Jansson et al., but has some major differences in architecture desing and skip connections.
+
+
+## References
+
+- Jansson, A., Humphrey, E., Montecchio, N., Bittner, R., Kumar, A., &Weyde, T. U-NET CONVOLUTIONAL NETWORKS. Proceedings of ISMIR, 2017
```
Comment (faroit) [inline submissions/JY1/description.md]: should be `and Weyde, T` instead of `&Weyde, T`

File: submissions/JY2/description.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+## Method
+
+Denoising auto-encoder with skip connections. It shares some similarities with the U-Net proposed by Jansson et al., but has some major differences in architecture desing and skip connections.
+
+In comparison to JY1 model, this one adds an enhancement module in addition to the main separation module.
+
+
+## References
+
+- Jansson, A., Humphrey, E., Montecchio, N., Bittner, R., Kumar, A., &Weyde, T. U-NET CONVOLUTIONAL NETWORKS. Proceedings of ISMIR, 2017
```
Comment (faroit) [inline submissions/JY2/description.md]: see above

File: submissions/JY3/description.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+## Method
+
+Denoising auto-encoder with skip connections. It shares some similarities with the U-Net proposed by Jansson et al., but has some major differences in architecture desing and skip connections.
+
+In conparison to the JY1 model, this one uses a different scheme of skip connections.
+
+
+## References
+
+- Jansson, A., Humphrey, E., Montecchio, N., Bittner, R., Kumar, A., &Weyde, T. SINGING VOICE SEPARATION WITH DEEP U-NET CONVOLUTIONAL NETWORKS. Proceedings of ISMIR, 2017
```
Comment (faroit) [inline submissions/JY3/description.md]: see above

**Comment by ciaua** [comment]
I have updated the three files. Thanks!

**Comment by faroit** [comment]
perfect. thanks

**Comment by faroit** [comment]
Due to popular demand, we allow to update pull requests until 3rd of April 23:59 UTC±00:00 before we merge it.

**Pull request merged**